### PR TITLE
VDH : small corrections and remarks

### PR DESCRIPTION
**Pull request by vdh-anssi**

### CODE REVIEW

File: src/en/07_ffi.md [hunk 1]
```diff
@@ -471,7 +471,7 @@ environment.
 > ### Recommendation {{#check FFI-NOENUM | Do not use incoming Rust `enum` at FFI boundary}}
 >
 > In a secure Rust development, when interfacing with a foreign language,
-> the Rust code should not accept incoming values from Rust `enum` types.
+> the Rust code should not accept incoming values of type Rust `enum`.
```
Comment (polazarus) [inline src/en/07_ffi.md]: `enum` is not a Rust type, more like a Rust kind.
Maybe "of any Rust `enum` type"?

File: src/en/07_ffi.md [hunk 2]
```diff
@@ -304,16 +304,6 @@ pointers are preferred over Rust references when binding to another language.
 On the one hand, reference types are very non-robust: they allow only pointers
 to valid memory objects. Any deviation leads to undefined behavior.
 
-> ### Rule {{#check FFI-CKREF | Do not use unchecked foreign references}}
->
-> In a secure Rust development, every foreign references that is transmitted to
-> Rust through FFI must be **checked on the foreign side** either automatically
-> (for instance, by a compiler) or manually.
->
-> Exceptions include Rust references in an opaque wrapping that is
```
Comment (polazarus) [inline src/en/07_ffi.md]: Exceptions were not moved with the rule.

File: src/en/02_devenv.md [hunk 3]
```diff
@@ -129,19 +129,19 @@ using the debug profile that normally enables runtime checks (for example
...
 a given system. This can be very useful, but it may also be difficult to know
-and remember at a given time all of the options that are effectively used, and
+and remember at a given time all the options that are effectively used, and
 in particular passed to the compiler. At the end, this can affect the confidence
 and robustness of the build process. It is preferable to centralize compiler
 options and flags in the configuration file `Cargo.toml`. For the case of
-environment variable `RUSTC_WRAPPER`, for example to generate part of code or
-to run external tools before Rust compilation, it is preferable to use the Cargo
-build scripts feature.
+environment variable `RUSTC_WRAPPER`, for example, that may is used to generate
```
Comment (polazarus) [inline src/en/02_devenv.md]: is -> be

**Comment by ad-anssi** [comment]
Thank you for your comments ! I've added a commit to take into account @polazarus corrections and merged.

**Pull request closed without merging**